default.nix

```diff
@@ -216,8 +216,8 @@ let overrideCabal = pkg: f: if pkg == null then null else haskellLib.overrideCab
         # Reflex packages
         ########################################################################
         reflex = dontHaddock (addFastWeakFlag (addReflexTraceEventsFlag (addReflexOptimizerFlag (self.callPackage (hackGet ./reflex) {}))));
-        reflex-dom = addReflexOptimizerFlag (doJailbreak reflexDom.reflex-dom);
-        reflex-dom-core = addReflexOptimizerFlag (doJailbreak reflexDom.reflex-dom-core);
+        reflex-dom = addReflexOptimizerFlag (dontHaddock (doJailbreak reflexDom.reflex-dom));
+        reflex-dom-core = addReflexOptimizerFlag (dontHaddock (doJailbreak reflexDom.reflex-dom-core));
```

Comment:
Uh-oh. The ability to hoogle the reflex-dom(-core) haddocks is pretty important. What's the issue with haddocks here?

Reply:
Yeah I'm not sure why haddock is broken - I'm using https://github.com/reflex-frp/reflex-dom/commit/00cdc8358052237625b39c40e807f0892fe0e682 so it seems like it should work. Is it possible they are just broken from ghcjs?